docs/contributing/introduction-to-contributing.md

```diff
@@ -115,52 +77,91 @@ GitHub supports [draft pull requests](https://github.blog/2019-02-14-introducing
 
 - _[3 tips for getting your Pull Request reviewed on GitHub](https://youtu.be/cuMeC-eZJJ4)_
 
-## Commits
+## Getting Started
 
-When contributing to OpenSauced projects, you can use this command to follow our conventional commits:
+### Setting Up Projects Locally
+
+1. [Fork](https://docs.github.com/en/get-started/quickstart/fork-a-repo) the repository you're working on from [github/open-sauced](https://github.com/open-sauced) to your own GitHub account.
+2. Clone the forked repository to your local machine.
+3. Check the project's README to determine how to run the project locally.
+
+For many of our repositories, you'll find the directions below work for running the project locally:
+
+Run `npm ci` to install the dependencies and set up the project.
+
+You can also use the shell commands below to get started once you have forked the repository:
```

Comment:
I need your thoughts. I'm thinking to make this a subsection (not sure about the title, though) or at least, adjust the wordings as below:

 ```suggestion
 #### Additional Information

For many of our repositories, you'll run `npm ci` to install the dependencies and set up the project locally.

You can also use the shell commands below to get started once you have forked the repository:
```

Reply:
I would suggest adding a sentence explaining what `npm ci` is because a lot people will probably only be used to `npm i` if working on personal projects before they got involved with open source work.

Reply:
I think the subsection works fine